tests/slow/src/test/scala/tests/mill/MillLspSuite.scala

```diff
@@ -141,6 +141,8 @@ class MillLspSuite extends BaseImportSuite("mill-import") {
            |  def scalaVersion = "${V.scala213}"
            |  def scalacOptions = Seq("-Xfatal-warnings", "-Ywarn-unused")
            |}
+           |/.mill-version
+           |${V.millVersion}
```

Comment:
For newest mill version this returns a result that additionally has a bunch of local symbols. Not sure why that is, but it is a separate issue.